cirq/contrib/qcircuit/optimizers.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2018 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import itertools
+
+from typing import Callable, Optional
+
+from cirq import circuits, ops
+
+
+def EnsureFinalMomentEmpty(circuit: circuits.Circuit) -> None:
```

Comment:
Use `snake_case` for method names.

Doc string.

Reply:
Done. (This was a vestige of the old optimizer class.)